source/Calamari/Kubernetes/Conventions/GatherAndApplyRawYamlConvention.cs

```diff
@@ -19,16 +19,16 @@ public class GatherAndApplyRawYamlConvention: IInstallConvention
     {
         private readonly ILog log;
         private readonly ICalamariFileSystem fileSystem;
-        private readonly ICommandLineRunner commandLineRunner;
+        private readonly Kubectl kubectl;
 
         public GatherAndApplyRawYamlConvention(
             ILog log,
             ICalamariFileSystem fileSystem,
-            ICommandLineRunner commandLineRunner)
+            Kubectl kubectl)
```

Comment:
Kubectl is now passed in as a constructor level parameter so that we only have a single instance. If we have multiple instances then each one will print out logs saying it has successfully being setup which doesn't look too good in the task log.